### PR TITLE
added 1st assignment

### PR DESCRIPTION
**Pull request by jamge**

### CODE REVIEW

**Review by jarandeshubham — APPROVED** [review]

File: balaji_jamge/assignment1/html1.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+<!DOCTTYPE html>
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html1.html:2]: remove extra t from doctype

File: balaji_jamge/assignment1/html1.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<html>
+<head> 
+
+<title> </title>
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html1.html:6]: add title

File: balaji_jamge/assignment1/html1.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+</head>
+
+<body>
+    <h1>HTML </h1>
+<h2>introduction</h2>
+    <h6> HTML is standard markup language for web page</h6>
+</body>
+</html>
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html1.html:14]: add new line

File: balaji_jamge/assignment1/html2.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <h2> HTML editor</h2>
+    <h4> Notepad or TextEdit.</h4>
+    <p> web pages can be created and modified by using professional HTML editor.<br>
+    Howeever,for learning HTML we recommend a simple text editor like NOtpad (PC) or TextEdit(Mac).</p>
+</body>
+
+
+
+
+
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html2.html:19]: remove extra lines

File: balaji_jamge/assignment1/html2.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+</html>
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html2.html:21]: add new line

File: balaji_jamge/assignment1/html3.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    <h4> Notepad or TextEdit.</h4>
+    <p> web pages can be created and modified by using professional HTML editor.<br>
+    Howeever,for learning HTML we recommend a simple text editor like NOtpad (PC) or TextEdit(Mac).</p>
+</body>
+
+
+
+
+
+
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html3.html:21]: remove extra lines

File: balaji_jamge/assignment1/html3.html [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+</html>
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html3.html:22]: add new line

File: balaji_jamge/assignment1/html4.html [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+        </title>
+    </head>
+    <body>
+        <h1>I Am Balaji Vitthalrao Jamge <h1>
+
+        <h2>I Am Basically Nanded but Currently Staying in Pune, 
+I Did My Post-Graduation Completed in M.Sc. Computer Science from Swami Ramanand Teertha Marathwada University Nanded
+        </h2>
+    </body>
+</html>
```
Comment (Pawan1005) [inline balaji_jamge/assignment1/html4.html:15]: add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Pull request merged**